packages/smarthr-ui/src/components/RadioButtonPanel/RadioButtonPanel.tsx

```diff
@@ -45,10 +80,37 @@ export const RadioButtonPanel: FC<Props> = ({ onClick, as, className, ...props }
     innerRef.current?.click()
   }, [])
 
+  const descriptionId = useId()
+  const actualAriaDescribedby = useMemo(
+    () =>
+      [children && descriptionId, ariaDescribedby].reduce((acc: string, str) => {
+        if (!str) {
+          return acc
+        }
+        return `${acc} ${str}`
+      }, ''),
+    [children, descriptionId, ariaDescribedby],
+  )
+
   return (
     // eslint-disable-next-line smarthr/a11y-delegate-element-has-role-presentation
-    <Base padding={1} role={role} onClick={handleOuterClick} as={as} className={actualClassName}>
-      <RadioButton {...props} ref={innerRef} />
+    <Base padding={1} role={role} onClick={handleOuterClick} as={as} className={classNames.base}>
+      <RadioButton
+        {...props}
+        ref={innerRef}
+        aria-describedby={actualAriaDescribedby}
+        className={classNames.radio}
+      >
+        <Cluster as="span">
+          {label}
+          {labelSuffix}
+        </Cluster>
```

Comment:
should: 微妙にズレてるので縦位置揃えたい

```suggestion
        <Cluster align="center" as="span">
          {label}
          {labelSuffix}
        </Cluster>
```